CHANGELOG.md

```diff
@@ -78,6 +78,13 @@
   - `bedtools/bedtools_merge`: Enhanced merging options and distance parameters
   - `bedtools/bedtools_sort`: Standardized sorting options
 
+* `bcftools`: Updated components to version 1.22 with comprehensive improvements including enhanced argument coverage, improved script patterns, biobox standard compliance, and comprehensive testing overhaul (PR #193):
+  * `bcftools_annotate`: Added `--verbosity` parameter; updated to use `meta_cpus` instead of `--threads` parameter
+  * `bcftools_concat`: Renamed `--compact_PS` to `--compact_ps`, `--remove_duplicates` to `--rm_dups`, `--min_PQ` to `--min_pq`; added `--remove_duplicates`, `--drop_genotypes`, `--verbosity`, `--write_index` parameters; updated to use `meta_cpus` instead of `--threads` parameter
```

Comment:
```suggestion
  * `bcftools_concat`: Renamed `--compact_PS` to `--compact_ps`, `--min_PQ` to `--min_pq`; added `--rm_dups`, `--drop_genotypes`, `--verbosity`, `--write_index` parameters; updated to use `meta_cpus` instead of `--threads` parameter
```